public/app/features/dashboard/components/PanelEditor/utils.ts

```diff
@@ -48,19 +48,38 @@ export const updateDefaultFieldConfigValue = (
       if (remove) {
         defaults.custom = omit(defaults.custom, name);
       } else {
-        defaults.custom = lodashSet({ ...defaults.custom }, name, value);
+        defaults.custom = setOptionImmutably({ ...defaults.custom }, name, value);
```

Comment:
probably just `defaults.custom` here, since we spread it internally now via `{ ... options }`